website/integrations/services/calibre-web/index.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Calibre-Web
+---
+
+<span class="badge badge--secondary">Support level: Community</span>
+
+## What is Calibre-Web
+
+From https://github.com/janeczku/calibre-web
+
+:::note
+Calibre-Web is a web app providing a clean interface for browsing, reading and downloading eBooks using a valid Calibre database.
+:::
+
+## Preparation
+
+The following placeholder will be used:
+
+-   `ldapservice` is the username of the authentik admin user you'd like to use for Calibre-Web to connect.
```

Comment:
Missing the FQDN of authentik, calibre and the ldap outpost (see https://goauthentik.io/integrations/services/jellyfin/#what-is-jellyfin)